src/man/firejail-profile.txt

```diff
@@ -295,7 +295,9 @@ Use the options no3d, nodvd, nosound, notv, nou2f and novideo for additional res
 Build a new /etc in a temporary
 filesystem, and copy the files and directories in the list.
 The files and directories in the list must be expressed as relative to
-the /etc directory.
+the /etc directory, and must not contain the / character
+(e.g., /etc/foo must be expressed as foo, but /etc/foo/bar --
+expressed as foo/bar -- is disallowed).
```

Comment:
For reference: #3991

Reply:
Would you like me to amend the text to include the full list of disallowed chars and/or reference #3991?

Reply:
FWIW: I specifically mentioned `/` because it seemed the easiest way to say "you can only have a file or dir that's directly in `/etc` / `/opt` / `/srv` -- you can't have anything that's in a subdir of `/etc` / `/opt` / `/srv`.  If there's a better way to express that, I'm open to alternate wording.

Reply:
> Would you like me to amend the text to include the full list of disallowed chars and/or reference #3991?

No. When #3991 gets implemented we can update the manpage. And `Error: "foo;bar" is an invalid filename: rejected character: ";"` is one of the most helpful messages we have.